### DISCUSSION TITLE
Update Technitium

### DISCUSSION THREAD

**Discussion by Donmeusi** [tteck/Proxmox#1584] [Help Needed] [answered]
Hello,
I tried to update technitium dns (11.1 to 11.3). With the command:
"bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/technitiumdns.sh)" in the LXC Console. It shows "Updated Successfully", but nothing happens. The version is still the old one.

Does anyone have an idea? Thank you in advance.

**Comment by tteck** [comment, marked as answer]
Try executing this command `bash <(curl -fsSL https://download.technitium.com/dns/install.sh)` in the LXC console. Please let me know if it successfully updates so that I can change the script.

**Reply by Donmeusi** [reply]
It worked. Thanks a lot!
